src/backend/cdb/dispatcher/cdbgang.c

```diff
@@ -68,6 +68,8 @@ int			host_segments = 0;
  */
 int			ic_htab_size = 0;
 
+int 		qe_idx = 0;
```

Comment:
what difference with `qe_identifier` ?

Reply:
Thanks for reminding. qe_identifier is better, I will fix it soon.

Reply:
I took a closer look and found that qe_identifier is the QE number used in the entire plan. But what we need is the sequence number of QE in a gang. Do you know that the variable that exists now can determine the sequence number? So that we could avoid transfering a new param.

Reply:
I can't tell, you can take a look of `sliceIndex`.

Reply:
Thank you @yjhjstz , I would look into them.